code/modules/vending/wardrobes.dm

```diff
@@ -80,7 +80,8 @@
 					/obj/item/clothing/suit/apron/surgical = 4,
 					/obj/item/clothing/mask/surgical = 4,
 					/obj/item/clothing/gloves/color/latex/nitrile = 2, //SKYRAT EDIT ADDITION
-					/obj/item/clothing/suit/toggle/labcoat/hospitalgown = 4)	//SKYRAT EDIT ADDITION
+					/obj/item/clothing/suit/toggle/labcoat/hospitalgown = 4,	//SKYRAT EDIT ADDITION
+					/obj/item/storage/belt/medbandolier = 4)	// SKYRAT EDIT ADDITION
```

Comment:
I'd probably make it only two, and have them be either something you can also make out of leather, or something along those lines, so it doesn't feel like mostly an upgrade from the medical belt for chemists, especially since I think that there's like, only two-three belts that spawn in the lockers, right?